### PR TITLE
tracepoints/operational: add gcc check for retpolines

### PR DESCRIPTION
**Pull request by jbastian**

### CODE REVIEW

File: tracepoints/operational/runtest.sh [hunk 1]
```diff
@@ -182,6 +182,32 @@ if [ $? -eq 0 ]; then
...
+if [ -e /sys/devices/system/cpu/vulnerabilities/spectre_v2 ]; then
+  CFLAGS=""
+  if [ "`uname -i`" = "x86_64" ]; then
+    CFLAGS="-mindirect-branch=thunk-extern"
+    CFLAGS+=" -mindirect-branch-register"
+    MITIGATION="retpoline"
+  elif [ "`uname -i`" = "s390x" ]; then
+    CFLAGS="-mindirect-branch=thunk-extern"
+    CFLAGS+=" -mindirect-branch-table "
+    CFLAGS+=" -mfunction-return=thunk-extern "
```
Comment (rasibley) [inline tracepoints/operational/runtest.sh]: Thanks @jbastian ! Is the space needed at the end here ? Also do you have a test job ?

Comment (jbastian) [reply tracepoints/operational/runtest.sh]: Oops, I see I put both leading and trailing spaces. It doesn't hurt, but it does look ugly. I'll clean that up.

Test job:
https://beaker.engineering.redhat.com/jobs/3966563

**Comment by jbastian** [comment]
Patch is updated with trailing whitespace removed from those two lines.

File: tracepoints/operational/runtest.sh (hunk 1 above)
Comment (jbastian) [reply tracepoints/operational/runtest.sh]: The patch is updated with the trailing whitespace removed from those two CFLAGS lines.

Comment (rasibley) [reply tracepoints/operational/runtest.sh]: @jbastian it was total nit-pick, looks good! Thanks for the test jobs to see it in action :-)

**Pull request merged**